src/components/Articles.jsx

```diff
@@ -1,23 +1,22 @@
 import { useEffect } from "react"
 import { useState } from "react"
 import ArticleCard from "./ArticleCard"
-import ApiRequests from "../ApiRequests/ApiRequests"
+import ApiRequests from "../Utils/ApiRequests"
 
 const Articles = () => {
     const [articles, setArticles] = useState("")
     const [loading, setLoading] = useState(true);
 
     useEffect(() => {
         ApiRequests.getArticles()
-        .then(({data : {articles}}) => {
+        .then((articles ) => {
```

Comment:
I like that this is now de-structured in the api file - meaning that by the time we get back up into our then block, it's really readable that what we're dealing with is just the array of articles.